src/dashboard/components/options/OptionsHome.js

```diff
@@ -139,34 +141,37 @@ export default function OptionsHome(props) {
             </Button>
           </p>
         </div>
-        <hr />
-        <h1 className="secondary-text">{t('Coming Soon!')}</h1>
-        <div className="dashboard__options__item disabled">
-          <h4>{t('Auto-Save')}</h4>
-          <Switch
-            disabled
-            isOn={!!settings.user.autoSave || true}
-            handleToggle={() => saveSetting('user.autoSave', !settings.user.autoSave)}
-            labelText={t('By default, use auto-save for projects')}
-          />
-        </div>
-        <div className="dashboard__options__item disabled">
+        <div className="dashboard__options__item">
           <h4>{t('Backup Location')}</h4>
           <FormGroup controlId="backupLocation">
             <ControlLabel>{t('Folder where backups are stored')}</ControlLabel>
-            <FormControl type="text" value={backupLocation} onChange={() => {}} disabled />
+            <p>{backupLocation}</p>
+            <Button bsSize="small" onClick={onChangeBackupLocation}>
+              {t('Choose...')}
+            </Button>
           </FormGroup>
         </div>
-        <div className="dashboard__options__item disabled">
+        <div className="dashboard__options__item">
           <h4>{t('Days of Backup')}</h4>
           <FormGroup controlId="backupDays">
-            <select onChange={(event) => setBackupType(event.target.value)} disabled>
+            <select onChange={(event) => setBackupType(event.target.value)}>
```

Comment:
This state is internal to the component.  i.e. if we refresh the page, the choice reverts to its default(!)
Again, I'm not sure that the state's necessary here.
We should use and alter the value in the settings store directly.